packages/ui/src/__mocks__/logto.tsx

```diff
@@ -0,0 +1,2 @@
+export const AppLogo = 'https://avatars.githubusercontent.com/u/88327661?s=200&v=4';
+export const AppContext = 'Build user identity in a modern way';
```

Comment:
should we just use `camelCase` for constants

Reply:
ah yes.